### PR TITLE
rpc: refacter to use const

### PR DESCRIPTION
**Pull request by s7v7nislands**
refactor to use const, it is more readable.

### CODE REVIEW

File: cmd/geth/config.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"reflect"
 	"unicode"
 
+	"github.com/ethereum/go-ethereum/rpc"
```
Comment (rjl493456442) [inline cmd/geth/config.go:27]: Can you merge these two with other external imports in the same block?

File: rpc/client.go [hunk 2]
```diff
@@ -436,13 +436,7 @@ func (c *Client) Notify(ctx context.Context, method string, args ...interface{})
 
 // EthSubscribe registers a subscription under the "eth" namespace.
 func (c *Client) EthSubscribe(ctx context.Context, channel interface{}, args ...interface{}) (*ClientSubscription, error) {
-	return c.Subscribe(ctx, "eth", channel, args...)
-}
-
-// ShhSubscribe registers a subscription under the "shh" namespace.
-// Deprecated: use Subscribe(ctx, "shh", ...).
```
Comment (rjl493456442) [inline rpc/client.go:443]: Since whisper is already deprecated and we don't have `shh` api exposed, I think it's safe to delete it now.

**Review by rjl493456442 — APPROVED** [review]
lgtm

File: cmd/geth/config.go (hunk 1 above)
Comment (s7v7nislands) [reply cmd/geth/config.go]: fix

**Comment by fjl** [comment]
I'm against this change. Package rpc is a generic JSON-RPC client/server, it should not know about specific Ethereum API names. Also, using constants instead of plain names just seems to add more code with very little benefit. The API names are already very compact and easy to remember.

**Comment by s7v7nislands** [comment]
@fjl how to find all "eth_xxx" easily? when using plain names, it is not easy to find all functions.

**Comment by fjl** [comment]
About the existing constant definitions in package rpc: The `EngineApi` definition should be removed. The `MetadataApi` constant should not be exported.

Regarding your second question

> how to find all "eth_xxx" easily? 

We an idea to add introspection in #25134, but it didn't work super well. I'd be open to adding a Go method on `rpc.Server` that returns all available method names though.

**Comment by s7v7nislands** [comment]
Sorry for my poor English, I mean when reading code how to easily find the function implemented because different eth_xx methods are defined in different API structure
By the way, getting the all methods in API is a nice feature.

**Pull request closed without merging**